modules/rgbd/src/hash_tsdf.cpp

```diff
@@ -16,10 +16,26 @@
 #include "opencv2/core/utils/trace.hpp"
 #include "utils.hpp"
 
+#define USE_INTERPOLATION_IN_GETNORMAL 1
+
 namespace cv
 {
 namespace kinfu
 {
+
+static inline TsdfType floatToTsdf(float num)
+{
+    //CV_Assert(-1 < num <= 1);
+    int8_t res = int8_t(num * (-128));
+    res = res ? res : (num < 0 ? 1 : -1);
+    return res;
+}
+
+static inline float tsdfToFloat(TsdfType num)
+{
+    return float(num) * (-0.0078125f); // * (-1/128)
```

Comment:
It's not necessary to calculate the constant; keeping it as `(-1.f/128.f)` is more human-readable and compiles in the same constant.

Reply:
done